src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import embed from 'vega-embed';
+export default function TechStackGraph() {
+  // let url = `${process.env.REACT_APP_DS_API_URL}/graph/tech-stack-by-role`;
+  let url =
+    'http://underdog-devs-ds-a-dev.us-east-1.elasticbeanstalk.com/graph/tech-stack-by-role';
+  // embed is an asynchronous api call, resolves in a promise
+  // for more information visit https://github.com/vega/vega-embed
+  useEffect(() => {
```

Comment:
Great use of useEffect to responsively update the tech stack info every time the webpage is refreshed. I'm sure that once the graph information is more fleshed out that the lorem ipsum paragraph will be updated with relevant info and change based on the graph.